examples/wget/Kconfig

```diff
@@ -12,11 +12,18 @@ config EXAMPLES_WGET
 
 if EXAMPLES_WGET
 
+config EXAMPLES_WGET_STACKSIZE
+	int "wget stack size"
+	default 2048
+	---help---
+		Stack size of "wget" command. If you use SSL connection,
+		you should increase this value around 8096.
+
 config EXAMPLES_WGET_URL
```

Comment:
should we remve this option? since it get from command line now.

Reply:
Is it possible to use both of command line and option? I found it used in some defconfigs.

Reply:
But your patch don't reference EXAMPLES_WGET_URL anymore and the code return -1 if user don't pass a URL. So please make your code and Kconfig consistent with each other.

Reply:
Sorry, my previous comment was mistake.

I know this patch is wrong.
I think it has 2 ways for fix it. 1) completely remove EXAMPLES_WGET_URL option and 2) to be able to use both of command line and option.
Do you think which better ones?

Reply:
Both are fine for me.

Reply:
Thanks. I'll fix it to both.